src/main/java/org/jenkinsci/plugins/authorizeproject/GlobalQueueItemAuthenticator.java

```diff
@@ -0,0 +1,49 @@
+package org.jenkinsci.plugins.authorizeproject;
+
+import hudson.Extension;
+import hudson.model.Job;
+import hudson.model.Queue;
+import jenkins.security.QueueItemAuthenticator;
+import jenkins.security.QueueItemAuthenticatorDescriptor;
+import org.acegisecurity.Authentication;
+import org.jenkinsci.plugins.authorizeproject.strategy.SystemAuthorizationStrategy;
+import org.kohsuke.stapler.DataBoundConstructor;
+
+/**
+ * A global default authenticator to allow changing the default for all projects.
+ *
+ * @since 1.1.1
+ */
+public class GlobalQueueItemAuthenticator extends QueueItemAuthenticator {
+    private final AuthorizeProjectStrategy strategy;
+
+    @DataBoundConstructor
+    public GlobalQueueItemAuthenticator(AuthorizeProjectStrategy defaultStrategy) {
+        this.strategy = defaultStrategy;
+    }
+
+    public AuthorizeProjectStrategy getStrategy() {
+        return strategy;
+    }
+
+    @Override
+    public Authentication authenticate(Queue.Item item) {
+        return item.task instanceof Job ? strategy.authenticate((Job<?, ?>) item.task, item) : null;
+    }
+
+    @Extension
+    public static class DescriptorImpl extends QueueItemAuthenticatorDescriptor {
+        /**
+         * {@inheritDoc}
+         */
+        @Override
+        public String getDisplayName() {
+            return Messages.GlobalQueueItemAuthenticator_DisplayName();
+        }
+
+        public AuthorizeProjectStrategy getDefaultStrategy() {
+            return new SystemAuthorizationStrategy();
+        }
```

Comment:
If we don't provide a default then it defaults to "Run as specific user"... and that somehow lets you specify an empty user... so you can very easily end up with
![screen shot 2016-02-08 at 12 00 44](https://cloud.githubusercontent.com/assets/209336/12885167/abada59a-ce5b-11e5-9d98-56e850275254.png) and then it's just a short hop, skip and jump to 
![screen shot 2016-02-08 at 12 09 28](https://cloud.githubusercontent.com/assets/209336/12885304/e0ab7ece-ce5c-11e5-8bd7-ffb05a84e15d.png)

So I think we do need to specify a default... however thinking some more I can accept a default of `AnonymousAuthorizationStrategy` as that fits with secure by default mode... in other words, the user adding this needs to take a deliberate decision to use the `SystemAuthorizationStrategy`